hibernate-core/src/main/resources/org/hibernate/xsd/mapping/legacy-mapping-4.0.xsd

```diff
@@ -119,11 +119,13 @@
         </xs:sequence>
         <xs:attribute name="condition" type="xs:string"/>
         <xs:attribute name="name" use="required" type="xs:string"/>
+        <xs:attribute name="auto-enabled" type="xs:boolean" default="false"/>
```

Comment:
Please revert the changes to this file. hbm.xml is deprecated and will most likely be removed in 7.0.